packages/orama/src/methods/serialization.ts

```diff
@@ -3,16 +3,19 @@ import { Orama } from '../types.js'
 export interface RawData {
   index: unknown
   docs: unknown
+  sort: unknown
```

Comment:
Non-blocking: I would probably call this `sorting` since so far we have 1-1 mapping.